spacy_udpipe/language.py

```diff
@@ -230,6 +231,12 @@ def __init__(
         self._path = None
         self._optimizer = None
 
+    def pipe(self, texts, **kwargs):
+        warnings.warn("Multiprocessing is not supported!")
+        if "n_process" in kwargs:
+            kwargs["n_process"] = 1
```

Comment:
Can you dump your investigation somewhere to outline why multiprocessing is not supported and what would it take to get there? You did a great job investigating, just don't want to lose that.